xt_training/runner.py

```diff
@@ -1,10 +1,13 @@
 import torch
 import numpy as np
 import json
-from collections import Iterable
+import logging
```

Comment:
This isn't used is it?

Reply:
It is used to check if x and y are lists. Also, in the detach objects function where we need to check if it is a list of tensors.

Reply:
I assume he was talking about logging hahah